src/api/app/controllers/projects/public_key_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Projects
```

Comment:
I would like to discuss controller modularity more broadly before introducing this into the codebase...